vms/platformvm/txs/executor/helpers_test.go

```diff
@@ -215,6 +215,8 @@ func addSubnet(
 
 	stateDiff.AddTx(testSubnet1, status.Committed)
 	require.NoError(stateDiff.Apply(env.state))
+	require.NoError(env.state.Commit())
+	defaultBalance -= env.config.GetCreateSubnetTxFee(env.clk.Time())
```

Comment:
it's probably better not to modify any defaults. otherwise you might have some unexpected outcomes in subsequent tests.

Reply:
You are right in principle, but current arrangement is wrong.
Depending on which fork a test activates, createSubnetTxFee may be not zero.
Current tests assume subnet creation fee is zero and don't scale down the default balance. These tests pass because the forkTime they set is not really coherent with the fork they intend to test.
This change here is a step in fixing this issue and I really need that for Durango UTs.
There is a different [thread of PRs](https://github.com/ava-labs/avalanchego/issues/2453) where this issue is handled more consistently. This is the bare minimum I need to have significant Durango UTs